src/shared/config.py

```diff
@@ -29,11 +28,16 @@ class Config(BaseModel):
     output: OutputConfig
     podcasts: Dict[str, str]
     processing: ProcessingConfig
-    remote_whisper: bool = False
     server: Optional[str] = None
     server_port: int = 5001
     threads: int = 1
-    whisper_model: str = "base"
+    remote_whisper: bool = False
+    whisper_api_key: Optional[str] = None
+    whisper_base_url: Optional[str] = None 
+    remote_whisper_model: str = "whisper-1" # openai model, use your own maybe
+    whisper_language: str = "en"
+    faster_whisper_server: bool = False # for quirks specific to the faster whisper server
+    whisper_model: str = "base" # for local whisper
```

Comment:
can we make this into a `WhisperConfig` subconfig similar to `ProcessingConfig` or `OutputConfig` above? Then, we can pass just that part to `RemoteWhisperTranscriber` when we initialize it. 

Even cooler (though very much not necessary and not 100% sure it works) would be something like 

```python
class LocalWhisperConfig:
  whisper_model: str = "base"

class RemoteWhisperConfig:
  whisper_base_url: str
  ...


class Config:
  ...
  whisper_config: RemoteWhisperConfig | LocalWhisperConfig

```

Reply:
I'd originally considered it, but I didn't know how to do that without breaking older configs. I can take a pass but it's going to be a while before I can.

Reply:
That's a good point about backwards compatibility. Maybe best to leave it as you have it.